org_status/org_status.py

```diff
@@ -83,18 +85,32 @@ def main():
 
     styled = (lambda l, *_: l) if args.no_color else colored
     verbose = print if args.verbose else (lambda *_: None)
+    hosts_only_print = print if args.hosts_only else verbose
+
+    # check only org status
+    if len(args.orgs) == 0 and not args.hosts_only:
+        print(styled('No organizations to check', 'red'))
+    elif len(args.orgs) == 0:
+        args.orgs.append('coala')
```

Comment:
Likewise, this should happen when `len(args.orgs) == 0 and not args.hosts_only`.